utils/log.go

```diff
@@ -68,3 +72,15 @@ func Errorf(format string, args ...interface{}) {
 func Debug() bool {
 	return logLevel == LogLevelDebug
 }
+
+func init() {
```

Comment:
Is it possible to write a test for that? Or is `init` executed before we can set an env variable in the tests?